integreat_cms/cms/templates/languages/region_list_section/region_list.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+{% load static %}
+{% block content %}
+    <div class="table-listing">
+        <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
```

Comment:
```suggestion
        <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white text-center">
```
Forgot, I think this makes it a little nicer still to look at.